app.rb

```diff
@@ -1,9 +1,30 @@
 require 'sinatra/base'
+require 'sinatra/reloader'
+require 'pg'
+require './lib/peep'
 
 class Chitter < Sinatra::Base
+  configure :development do
+    register Sinatra::Reloader
+  end
+
   get '/test' do
```

Comment:
It's good to remove any extraneous code from your implementation before making a Pull Request - there'll be lots of testing and exploration as you work on something, but it tidies it up to remove anything that is no longer needed for the program to meet its specification.